diff --git b/go.mod a/go.mod
index c2e7110..1c2892d 100644
--- b/go.mod
+++ a/go.mod
@@ -1,4 +1,4 @@
-module github.com/terraform-providers/terraform-provider-mailgun
+module github.com/wgebis/terraform-provider-mailgun
 
 go 1.19
 
diff --git b/main.go a/main.go
index e69df44..caab6a0 100644
--- b/main.go
+++ a/main.go
@@ -2,7 +2,7 @@ package main
 
 import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
-	"github.com/terraform-providers/terraform-provider-mailgun/mailgun"
+	"github.com/wgebis/terraform-provider-mailgun/mailgun"
 )
 
 func main() {
