From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Venelin <venelin@pulumi.com>
Date: Mon, 2 Dec 2024 17:10:58 +0000
Subject: [PATCH] fork


diff --git a/go.mod b/go.mod
index 59296c3..89ddbd5 100644
--- a/go.mod
+++ b/go.mod
@@ -1,4 +1,4 @@
-module github.com/terraform-providers/terraform-provider-mailgun
+module github.com/wgebis/terraform-provider-mailgun/mailgun
 
 go 1.24.1
 
diff --git a/main.go b/main.go
index e69df44..caab6a0 100644
--- a/main.go
+++ b/main.go
@@ -2,7 +2,7 @@ package main
 
 import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
-	"github.com/terraform-providers/terraform-provider-mailgun/mailgun"
+	"github.com/wgebis/terraform-provider-mailgun/mailgun"
 )
 
 func main() {
